### ISSUE TITLE
In admin when we add new user agent rule it show design issue

### ISSUE THREAD

**Issue by dipti2jcommerce** [magento/magento2#20480]
In admin when we add new user agent rule it show design issue

### Preconditions (*)

1.Magento verion 2.3
2.

### Steps to reproduce (*)

1.Go to admin configuration clicl on content ->Design ->Configuration edit design theme.
2.Add new user agent rule from Design Rule

### Expected result (*)
![screenshot-pr-17518 instances magento-community engineering-2019-01-22-19-44-51](https://user-images.githubusercontent.com/18118638/51541536-1562ff00-1e7f-11e9-9315-f9b6867582a6.png)

### Actual result (*)
![screenshot-magento23 local-2019-01-22-19-40-01](https://user-images.githubusercontent.com/18118638/51541529-12680e80-1e7f-11e9-8929-0193931e3048.png)

**Comment by magento-engcom-team** [comment]
Hi @dipti2jcommerce. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@dipti2jcommerce do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by priti2jcommerce** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @priti2jcommerce, here is your [Magento instance](https://i-20480-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-20480-2-3-develop.instances.magento-community.engineering/admin](https://i-20480-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by amol2jcommerce** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce, here is your [Magento instance](https://i-20480-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-20480-2-2-develop.instances.magento-community.engineering/admin](https://i-20480-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@priti2jcommerce Thank you for verifying the issue.

Unfortunately, not enough information was provided to created internal ticket. Please consider adding the following:
- [ ] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
@priti2jcommerce Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97877` were created

**Comment by sdzhepa** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [Magento instance](https://i-20480-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-20480-2-3-develop.instances.magento-community.engineering/admin](https://i-20480-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by engcom-Charlie** [comment]
Hello @dipti2jcommerce 

Thank you for contribution and collaboration!

We are not able to reproduce this issue on the lates `2.3-develop` branch by provided steps.

We are closing this issue due to _branch `2.2-develop` was closed and Magento no longer accepts pull requests and issues to the v2.2 release lines to focus all development efforts on v2.3._
 See [Accepted pull requests and ported code](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#rules) for more details

If you still faced this issue on `2.3` please create a new Issue with all required details according to [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines)
